app/edge/__module.js

```diff
@@ -0,0 +1,29 @@
+import angular from 'angular';
+
+angular.module('portainer.edge', []).config(function config($stateRegistryProvider) {
+  const edge = {
+    name: 'edge',
+    url: '/edge',
+    parent: 'root',
+    abstract: true,
+    onEnter: /* @ngInject */ function onEnter($state, Notifications, Authentication) {
```

Comment:
No need for extra logic around URL access for unauthenticated users as long as the API server is preventing unauthorized operations.

Reply:
I actually liked that :D